plutus-playground/deployment/terraform/loadbalancing.tf

```diff
@@ -0,0 +1,181 @@
+    lb_port            = 443
+    lb_protocol        = "https"
+    ssl_certificate_id = "${data.aws_acm_certificate.kevm_private.arn}"
+  }
+
+  health_check {
+    healthy_threshold   = 2
+    unhealthy_threshold = 2
+    timeout             = 3
+    target              = "HTTP:80/"
+    interval            = 30
+  }
+
+  instances                   = ["${aws_instance.playground_a.id}", "${aws_instance.playground_b.id}"]
+  cross_zone_load_balancing   = false
+  idle_timeout                = 400
+  connection_draining         = true
+  connection_draining_timeout = 400
+
+  tags {
+    Name        = "${var.project}_${var.env}_public_alb"
+    Project     = "${var.project}"
+    Environment = "${var.env}"
+  }
+}
+
+resource "aws_route53_record" "alb" {
+  zone_id = "ZBC2AQBA8QH4G"
+  name    = "${var.env}.plutus.iohkdev.io"
+  type    = "A"
+
+  alias {
+    name                   = "${aws_elb.plutus.dns_name}"
+    zone_id                = "${aws_elb.plutus.zone_id}"
+    evaluate_target_health = true
+  }
+}
+
+## Use this ALB config if we want to switch to an ALB
+
+#resource "aws_alb" "plutus" {
+#  subnets = ["${aws_subnet.public.*.id}"]
+#
+#  security_groups = ["${aws_security_group.public_alb.id}"]
+#  internal        = false
+#
+#  tags {
+#    Name        = "${var.project}_${var.env}_public_alb"
+#    Project     = "${var.project}"
+#    Environment = "${var.env}"
+#  }
+#
+#  # access_logs {
+#  #   bucket = "${var.s3_bucket}"
+#  #   prefix = "ELB_logs"
+#  # }
+#}
+#
+#resource "aws_alb_listener" "playground" {
+#  load_balancer_arn = "${aws_alb.plutus.arn}"
+#  port              = "443"
+#  protocol          = "HTTPS"
+#  certificate_arn   = "${data.aws_acm_certificate.kevm_private.arn}"
+#
+#  default_action {
+#    target_group_arn = "${aws_alb_target_group.playground.arn}"
+#    type             = "forward"
+#  }
+#}
+#
+#resource "aws_alb_target_group" "playground" {
+#  port     = "80"
+#  protocol = "HTTP"
+#  vpc_id   = "${aws_vpc.plutus.id}"
+#
+#  # health_check {
+#  #   path = "/healthcheck"
+#  # }
+#}
+#
+#resource "aws_alb_listener_rule" "playground" {
+#  depends_on   = ["aws_alb_target_group.playground"]
+#  listener_arn = "${aws_alb_listener.playground.arn}"
+#  priority     = 100
+#
+#  action {
+#    type             = "forward"
+#    target_group_arn = "${aws_alb_target_group.playground.id}"
+#  }
+#
+#  condition {
+#    field  = "path-pattern"
+#    values = ["*"]
+#  }
+#}
+#
+#resource "aws_alb_target_group_attachment" "playground_a" {
+#  target_group_arn = "${aws_alb_target_group.playground.arn}"
+#  target_id        = "${aws_instance.playground_a.id}"
+#  port             = "80"
+#}
+#
+#resource "aws_route53_record" "alb" {
+#  zone_id = "Z2YDMO25KPZVPP"
+#  name    = "${var.env}.plutus.iohkdev.io"
+#  type    = "A"
+#
+#  alias {
+#    name                   = "${aws_alb.plutus.dns_name}"
+#    zone_id                = "${aws_alb.plutus.zone_id}"
+#    evaluate_target_health = true
+#  }
+#}
```

Comment:
commented out code

Reply:
haven't made a decision on this, will look tomorrow

Reply:
going to wait for @disassembler to merge in his changes before deciding on this, I don't want to remove the comments in case I change to an ALB. I think we can merge as long as you trust me to resolve this later.

Reply:
If you want to store code that we're not currently using somewhere I'd generally prefer to do that in a branch rather than in a comment, but if you'd prefer to keep it here I don't mind. But then add a note in the code so we know to delete it later.

Reply:
there are comments, see lines 50 and 107. A branch will become out of date quickly at this point

Reply:
true but regardless I want to see the changes the @disassembler has made first.